How to correctly use local-down?

I was looking at a fix for https://github.com/tablelandnetwork/local-tableland/issues/48 and can't seem to get `make local-down` to work how I thought it would.

### What I've done

1. Start a hardhat evm node and deploy a registry contract to it.
2. In this repo run `cd docker && make local-up`.  The Validator starts correctly and seems to work as expected.
3. From a second terminal window in this repo run `cd docker && make local-down`.  This produces some warnings and an error, show below.

### What happens

```
WARN[0000] The "PLATFORM" variable is not set. Defaulting to a blank string. 
WARN[0000] The "BOOTSTRAP_BACKUP_URL" variable is not set. Defaulting to a blank string. 
WARN[0000] The "ENVIRONMENT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "ENVIRONMENT" variable is not set. Defaulting to a blank string. 
open /Users/jwagner/Workspaces/textile/github/go-tableland/docker//api/.env_validator: no such file or directory
make: *** [local-down] Error 14
```

Issue [#48](https://github.com/tablelandnetwork/local-tableland/issues/48) isn't really causing any problems and we could close it with `won't fix` if there isn't an easy solution for this, but wanted to make sure I'm not missing something obvious.

--- a/docker/Makefile
+++ b/docker/Makefile
@@ -1,11 +1,16 @@
 local-up:
+	touch local/api/.env_validator
 	PLATFORM=$(shell uname -m) \
 	ENVIRONMENT=local \
 	COMPOSE_DOCKER_CLI_BUILD=1 \
+	BOOTSTRAP_BACKUP_URL= \
 	docker compose -f docker-compose.yml -f local/docker-compose.override.yml up --build
 .PHONY: local-up
 
 local-down:
+	PLATFORM=$(shell uname -m) \
+	ENVIRONMENT=local \
+	BOOTSTRAP_BACKUP_URL= \
 	docker compose -f docker-compose.yml -f local/docker-compose.override.yml down
 .PHONY: local-down
 

